package ru.otus.otuskotlin.blog.backend.common.exceptions

object NoneException : Throwable()
